src/pages/Appointments/components/PractitionerSelector.tsx

```diff
@@ -193,7 +196,14 @@ export const PractitionerSelector = ({
     );
     onSelect(remainingSelected);
   };
-
+  const filteredPractitioners = useMemo(() => {
+    if (!organizationUsers?.users) return [];
+    const term = practitionerSearch.trim().toLowerCase();
+    if (!term) return organizationUsers.users;
+    return organizationUsers.users.filter((p) =>
```

Comment:
You need to handle child organizations as well. If I navigate into nested org, search wouldn't work.